open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/pom.xml

```diff
@@ -333,99 +343,6 @@
 
     <build>
         <plugins>
-            <plugin>
```

Comment:
For the consumers of the current jar (where I think the main question will be how to ensure the static files are served - maybe in an additional container)

See also /Users/jonesn/src/egeria/open-metadata-distribution/open-metadata-assemblies -- this is where this artifact is used to create a distribution archive, which we then use for our containers - specifically using the definitions in src/main/assemblies/egeria-omag.xml

The actual launch of the UI container for k8s is in open-metadata-resources/open-metadata-deployment/charts/odpi-egeria-lab/templates/egeria-ui.yaml & we have similar for compose in open-metadata-resources/open-metadata-deployment/compose/tutorials/egeria-tutorial.yaml